File: src/main/java/com/snackgame/server/member/controller/dto/GroupRequest.java
package com.snackgame.server.member.controller.dto;

import javax.validation.constraints.NotBlank;

import lombok.Getter;
import lombok.NoArgsConstructor;

@NoArgsConstructor
@Getter
public class GroupRequest {

    @NotBlank(message = "그룹 이름은 공백일 수 없습니다")
    private String group;
}

File: src/main/java/com/snackgame/server/member/service/dto/GroupRequest.kt
package com.snackgame.server.member.service.dto

import com.fasterxml.jackson.annotation.JsonCreator
import javax.validation.constraints.Pattern


data class GroupRequest @JsonCreator constructor(
    @Pattern(regexp = "[a-zA-Z가-힣0-9-_]{2,10}", message = "사용할 수 없는 그룹이름입니다")
    val group: String?
)
